ci-operator/step-registry/mco/conf/day2/destroy-custom-mcp/mco-conf-day2-destroy-custom-mcp-commands.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+set -e
+set -u
+set -o pipefail
+
+function set_proxy () {
+    if [ -s "${SHARED_DIR}/proxy-conf.sh" ]; then
+        echo "Setting the proxy ${SHARED_DIR}/proxy-conf.sh"
+        # shellcheck source=/dev/null
+        source "${SHARED_DIR}/proxy-conf.sh"
+    else
+        echo "No proxy settings"
+    fi
+}
+
+function run_command() {
+    local CMD="$1"
+    echo "Running command: ${CMD}"
+    eval "${CMD}"
+}
+
+# waiting for mcp become a given status
+function wait_mcp(){
+    local mcp_name="${1}" expected_updated="$2" expected_machine_count="$3"
+    local wait_time=30
+    while (( wait_time > 0 )); do
+        sleep 10
+        out="$(oc get machineconfigpools ${mcp_name} --no-headers)"
+        echo $out
+        updated="$(echo "${out}" | awk '{print $3}')"
+        MACHINECOUNT="$(echo "${out}" | awk '{print $6}')"
```

Comment:
Suggest to always declare these local env var, to avoid potential conflicts with other function using the same var name.